packages/material-ui/src/Tab/Tab.js

```diff
@@ -18,13 +18,11 @@ export const styles = (theme) => ({
     padding: '6px 12px',
     [theme.breakpoints.up('sm')]: {
       padding: '6px 24px',
+      minWidth: 160,
     },
```

Comment:
> What do you think about?
> - removing the min-width style
> - keeping the mobile & desktop padding at 24px

Like so?

```suggestion
    padding: '6px 24px',
```

I think that's fair (and it seems to match https://material-components.github.io/material-components-web-catalog/#/component/tabs and would fix #15824).